deployment/helm/templates/custom-configuration.yaml

```diff
@@ -27,6 +45,14 @@ spec:
             {{`"{{PLACEHOLDER}}"`}} : "1"
             memory: {{ .Values.custom.configuration.brokerPod.resources.memoryLimit }}
             cpu: {{ .Values.custom.configuration.brokerPod.resources.cpuLimit }}
+        {{- with .Values.custom.configuration.brokerPod.volumeMounts}}
```

Comment:
just curious, why use `with` here, but `range` for evnFrom?

Reply:
i tried to use range with `envFrom` as well but ran into issues since there is a higher level type specification (`secretRef` vs `configMapRef`), so it seemed better to be more specific in the chart